docs/source/testing-and-debugging.md

```diff
@@ -135,3 +135,37 @@ of the transaction.
         result, _ = await self.call("get_message", return_type=str)
         self.assertEqual(new_message, result)
 ```
+
+#### Accessing the Storage
+
+To get the key-value pairs stored in the smart contract's storage, you can use the `get_storage` method from 
+`SmartContractTestCase`. Use it with the prefix of the keys you want to access, and you'll get a `dict[bytes, bytes]`.
+If you want to get a dictionary of other types, you can also pass a function as argument to process the keys or values
+before adding it to the dict. If want to remove the prefix from the result you can use the `remove_prefix` parameter.
+
+```python
+    # inside the HelloWorldWithDeployTest class
+    async def test_smart_contract_storage(self):
+        from boaconstructor.storage import as_uint160, as_int
+        from typing import cast
+
+        # this is the prefix that the GAS smart contract uses to store the accounts and their balances
+        account_prefix = b'\x14'
+        
+        # the return from get_storage needs to be casted to the expected type
```

Comment:
just to be clear: it **does not** need to be casted, the values are going to be processed as long the user gives a `key_post_processor` or a `value_post_processor`.

the casting here is just for the IDE handle the types correctly because the method is annotated with `dict[bytes, bytes]` as its return type